### PR TITLE
--force option for slcli firewall add, firewall cancel command - issue #2032

### PR DESCRIPTION
**Pull request by JayasilanJibm**
Hi,

Fixes: #2032 

Please review this PR. 

Added force option for both slcli firewall add and cancel command.

Thanks,
Jayasilan

### CODE REVIEW

File: SoftLayer/CLI/file/list.py [hunk 1]
```diff
@@ -67,7 +67,11 @@
               default=','.join(DEFAULT_COLUMNS))
 @environment.pass_env
 def cli(env, sortby, columns, datacenter, username, storage_type, order):
-    """List file storage."""
+    """List file storage.
+
+    Example:
+            slcli file volume-list -d dal10 --storage-type endurance --sortby capacity_gb
+            """
```
Comment (allmightyspiff) [inline SoftLayer/CLI/file/list.py:74]:
```suggestion
    """List file storage.

    Example::
            slcli file volume-list -d dal10 --storage-type endurance --sortby capacity_gb
    """
```

File: SoftLayer/CLI/file/modify.py [hunk 2]
```diff
@@ -31,9 +32,19 @@
...
                    'must also be greater than 0.25.]',
               type=click.Choice(['0.25', '2', '4', '10']))
+@click.option('--force',  default=False, is_flag=True, help="Force modify")
 @environment.pass_env
-def cli(env, volume_id, new_size, new_iops, new_tier):
-    """Modify an existing file storage volume."""
+def cli(env, volume_id, new_size, new_iops, new_tier, force):
+    """Modify an existing file storage volume.
+
+    Examples:
```
Comment (allmightyspiff) [inline SoftLayer/CLI/file/modify.py:40]:
```suggestion
    Example::
```

File: SoftLayer/CLI/file/options.py [hunk 3]
```diff
@@ -19,7 +19,12 @@
                    'add it to the --prices option using location short name, e.g. --prices dal13')
 @environment.pass_env
 def cli(env, prices, location=None):
-    """List all options for ordering a block storage"""
+    """List all options for ordering a block storage
+
+    Examples:
```
Comment (allmightyspiff) [inline SoftLayer/CLI/file/options.py:24]:
```suggestion
    Example::
```

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Minor formatting changes, mostly just use `Example::` (with 2 :: at the end). Its a special syntax for the automatic documentation generator we use.

**Comment by ramkishor-ch** [comment]
Hi,

Code looks good.

Screenshot 2023-07-27 at 4 03 29 PM

Thanks,
Ramkishor Chaladi.

**Review by ramkishor-ch — APPROVED** [review]

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**